index.html

```diff
@@ -18,63 +18,73 @@
       <header class="header">
         <nav class="menu">
           <a href="#"><h2>My Logo</h2></a>
-          <a href="#"
+
+          <ul class="menu__links">
+            <li class="menu__links-item">Portofolio</li>
+            <li class="menu__links-item">About</li>
+            <li class="menu__links-item">Contact</li>
+          </ul>
+
+          <a class="humbeger" href="#"
```

Comment:
- [ ] Kindly fix all broken images and icons for a better UI(user interface). 😎

Reply:
Okay. What can  I modify from the navbar menu??